### PR TITLE
feat: add method to obtain the map bounds

### LINKED ISSUE #93: Get coordinate bounds.
Hey! Love the addon, super helpful for Vaadin 14 usage. I was wondering if it was possible to either listen for the `bounds_changed` event in google's maps api and then, using the `getBounds()` function, return the two coordinate bounds. I want to implement a functionality so I can see all the markers that are currently visible into my app. Is this possible to get added or is there some javascript events I can listen to?

I was doing some poking around inside of the source and was only able to listen to drag events, not much success.

### PR DESCRIPTION
**Pull request by paodb**
Close #93

### CODE REVIEW

File: src/main/java/com/flowingcode/vaadin/addons/googlemaps/LatLonBounds.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  public LatLonBounds() {}
+
+  public LatLonBounds(JsonObject bounds) {
+    setSouth(bounds.getNumber("south"));
+    setWest(bounds.getNumber("west"));
+    setNorth(bounds.getNumber("north"));
+    setEast(bounds.getNumber("east"));
+  }
+
+  public double getSouth() {
```
Comment (javier-godoy) [inline src/main/java/com/flowingcode/vaadin/addons/googlemaps/LatLonBounds.java:46]: Use lombok `@Getter` and `@Setter`

File: src/main/java/com/flowingcode/vaadin/addons/googlemaps/LatLonBounds.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  public double getEast() {
+    return east;
+  }
+
+  public void setEast(double east) {
+    this.east = east;
+  }
+
+  @Override
+  public String toString() {
```
Comment (javier-godoy) [inline src/main/java/com/flowingcode/vaadin/addons/googlemaps/LatLonBounds.java:79]: Annotate the class with`@ToString(includeFieldNames=true)` instead. See https://projectlombok.org/features/ToString

**Review by javier-godoy — CHANGES_REQUESTED** [review]

File: src/main/java/com/flowingcode/vaadin/addons/googlemaps/LatLonBounds.java (hunk 1 above)
Comment (paodb) [reply src/main/java/com/flowingcode/vaadin/addons/googlemaps/LatLonBounds.java:46]: Lombok is not being used in the add-on. No need to replace setters/getters.

File: src/main/java/com/flowingcode/vaadin/addons/googlemaps/LatLonBounds.java (hunk 2 above)
Comment (paodb) [reply src/main/java/com/flowingcode/vaadin/addons/googlemaps/LatLonBounds.java:79]: Lombok is not being used in the add-on.

**Review by mlopezFC — APPROVED** [review]
I approve this change. Regarding the lombok thing, we should create another ticket to discuss what to do.

**Pull request merged**